src/controller.rs

```diff
@@ -581,7 +604,7 @@ impl AppInstanceLike {
                             name: "setup-delete".to_string(),
                             // We need to use the bitnami image to make use of the in built
                             // shell to use the stdout redirection into a file.
-                            image: Some(apply::KUBECTL_IMAGE.to_string()),
+                            image: Some(ctx.apply_step_image()),
```

Comment:
I vaguely recall that this split came about from wanting to use the official `kubectl` image in the apply step, because we had no redirect needs there; however, I'm wondering whether the reasoning here needs to be stronger to justify it.

My proposal here would be to backtrack on that and simply use the `bitnami` image everywhere to simplify this. The reason being is that from a user perspective, I can't see any need to set different images for the steps. E.g. in an airgapped environment using a mirror of `bitnami/kubectl` would work exactly the same way anyway and it is simply "yet another image" to retrieve by having these split.

Reply:
@mkmik Any thoughts on this one?